src/users/dtos/userItems-pagination-query.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { IsOptional, IsInt, Min } from 'class-validator';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+
+export class UserItemsPaginationQueryDto {
+  @IsOptional()
+  @IsInt()
+  @ApiPropertyOptional({
+    description: '메인 카테고리 ID',
+    example: 1,
+  })
+  @Type(() => Number)
+  mainCategoryId?: number;
+
+  @IsOptional()
+  @IsInt()
+  @ApiPropertyOptional({
+    description: '서브 카테고리 ID',
+    example: 2,
+  })
+  @Type(() => Number)
+  subCategoryId?: number;
+
+  @IsInt()
+  @Min(1)
+  @ApiPropertyOptional({
+    description: '페이지 번호 (기본값: 1)',
+    example: 1,
+  })
```

Comment:
```suggestion
@ApiPropertyOptional({
    description: '페이지 번호 (기본값: 1)',
    example: 1,
    default: 1,
  })
```
default 옵션을 넣으면 기본값이 있다는걸 글로 설명하지 않아도 표현해 줄 수 있어요. 추가해보면 좋을거 같습니다.

Reply:
감사합니다 ㅎ